### DISCUSSION TITLE
Can I remove an email from my GitHub account but keep the commits on it?

### DISCUSSION THREAD

**Discussion by dungng1321** [community/community#45027] [Repositories]
### Select Topic Area

Question

### Body

I want to delete an old email addres. Can I remove an email from my GitHub account but keep the commits on it?

**Comment by radar-blip** [comment]
You could try it, and if it doesn't work, add it back again.

**Comment by seon22break** [comment]
Yes. Change the email from global settings in your pc. 
```
git config --global user.email johndoe@example.com
```
It is required for change email un yours commits.
